sdk/cognitiveservices/cognitiveservices-textanalytics/review/cognitiveservices-textanalytics.api.md

```diff
@@ -0,0 +1,346 @@
+## API Report File for "@azure/cognitiveservices-textanalytics"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ApiKeyCredentials } from '@azure/core-http';
+import * as coreHttp from '@azure/core-http';
+import { PipelineOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/identity';
+
+// @public (undocumented)
+export interface AnalyzeSentimentErrorResult extends TextAnalysisErrorResult {
+}
+
+// @public (undocumented)
+export interface AnalyzeSentimentOptions extends TextAnalyticsClientSentimentOptionalParams {
+}
+
+// @public (undocumented)
+export type AnalyzeSentimentResult = AnalyzeSentimentSuccessResult | AnalyzeSentimentErrorResult;
+
+// @public (undocumented)
+export interface AnalyzeSentimentResultCollection extends Array<AnalyzeSentimentResult> {
+    modelVersion: string;
+    statistics?: RequestStatistics;
+}
+
+// @public (undocumented)
+export interface AnalyzeSentimentSuccessResult extends TextAnalysisSuccessResult {
+    documentScores: number;
+    sentences: SentenceSentiment[];
+    sentiment: TextSentiment;
+}
+
+// @public
+export class CognitiveServicesCredentials extends ApiKeyCredentials {
+    constructor(subscriptionKey: string);
+}
+
+// @public
+export interface DetectedLanguage {
+    iso6391Name: string;
+    name: string;
+    score: number;
+}
+
+// @public (undocumented)
+export interface DetectLanguageErrorResult extends TextAnalysisErrorResult {
+}
+
+// @public (undocumented)
+export interface DetectLanguageOptions extends TextAnalyticsClientLanguagesOptionalParams {
+}
+
+// @public (undocumented)
+export type DetectLanguageResult = DetectLanguageSuccessResult | DetectLanguageErrorResult;
+
+// @public (undocumented)
+export interface DetectLanguageResultCollection extends Array<DetectLanguageResult> {
+    modelVersion: string;
+    statistics?: RequestStatistics;
+}
+
+// @public (undocumented)
+export interface DetectLanguagesOptions extends TextAnalyticsClientLanguagesOptionalParams {
+}
+
+// @public (undocumented)
+export interface DetectLanguageSuccessResult extends TextAnalysisSuccessResult {
+    readonly detectedLanguages: DetectedLanguage[];
+    readonly primaryLanguage: DetectedLanguage | undefined;
+}
+
+// @public
+export interface DocumentStatistics {
+    charactersCount: number;
```

Comment:
Check python, see if they have a count suffix. If so, we can too.

Reply:
Python leaves it as `characters_count` and `transactions_count`: https://github.com/kristapratico/azure-sdk-for-python/blob/textanalytics/sdk/cognitiveservices/azure-cognitiveservices-language-textanalytics/azure/cognitiveservices/language/textanalytics/models/document_statistics.py

@kristapratico thoughts on making the first part singular instead of collective?

Reply:
Agree that the singular sounds better

Reply:
We moved it to singular in the .NET API as well.